src/fixtures/assets.js

```diff
@@ -42,8 +41,6 @@ export const initialAssets = [
     for your digital life.`,
     name: 'Pillar',
     symbol: 'PLR',
-    wallpaperUrl: 'assets/images/tokens/wallpaper/wp_blue.png',
```

Comment:
I can still see the reference for `wallpaperUrl` and `iconUrl` in the code. I would just remove the values, or do a bigger code clean up